doc/MANUAL.md

```diff
@@ -1080,6 +1080,10 @@ This option specifies the minimum EPSS percentile of CVE range between 0 to 100
 
 This option specifies the minimum EPSS probability of CVE range between o to 100 to report. The default value is 0 which result in all CVEs being reported.
 
+### Automatic Metrics Activation
+
+If either `--epss-percentile` or `--epss-probability` is set, the system will automatically enable the `--metrics` option. This eliminates the need for users to explicitly set `--metrics` when utilizing `--epss-percentile` or `--epss-probability`. This streamlines the process for users, eliminating the need for manual `--metrics` configuration.
```

Comment:
```suggestion
If either `--epss-percentile` or `--epss-probability` is set, the system will automatically enable the `--metrics` option so that the epss data will be loaded and displayed.
```

Making this shorter and focusing on *why* rather than *what* is being done here.